src/ElectricPokemon.java

```diff
@@ -0,0 +1,125 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+
+    private static final String type = "electric";
+    private static final List<String> attacks = Arrays.asList("ThunderPunch", "ElectroBall", "Thunder", "VoltTackle");
+
+    public ElectricPokemon(String name, int level, int hp, String food, String sound) {
+        super(name, level, hp, food, sound, type);
+    }
+
+    void voltTackle(Pokemon name, Pokemon enemy) {
+        System.out.println(name.getName() + " used Volt Tackle on " + enemy.getName());
+        switch (enemy.getType()) {
+            case "water":
+                System.out.println(enemy.getName() + " loses 40 HP");
+                enemy.setHp(enemy.getHp() - 40);
+                break;
+
+            case "grass":
+                System.out.println(enemy.getName() + " loses 30 HP");
+                enemy.setHp(enemy.getHp() - 30);
+                break;
+
+            case "fire":
+                System.out.println(enemy.getName() + " loses 20 HP");
+                enemy.setHp(enemy.getHp() - 20);
+                break;
+
+            default:
+                System.out.println(enemy.getName() + " loses 10 HP");
+                enemy.setHp(enemy.getHp() - 10);
+
+        }
+        System.out.println(enemy.getName() + " has " + enemy.getHp() + " HP left.");
+    }
+
+    void thunder(Pokemon name, Pokemon enemy) {
+        System.out.println(name.getName() + " used Thunder on " + enemy.getName());
+        switch (enemy.getType()) {
```

Comment:
al deze switch statments zijn wel veel van hetzelfde. Het zou mooi zijn als je dit op een of andere manier kan extraheren in een helper methode, zodat je het amar 1 keer hoeft te typen